rtsyntax.php

```diff
@@ -1,68 +1,108 @@
-
-            add_filter( 'mce_external_plugins', array( $this, 'rtsyntax_buttons' ) );
-            add_filter( 'mce_buttons', array( $this, 'register_rtsyntax_buttons' ) );
-        } else {
-            add_action( 'wp_enqueue_scripts', array( $this, 'enqueue' ) );
-            add_action( 'wp_head', array( $this, 'onload' ) );
-            add_action( 'the_content', array( $this, 'convert_pres' ) );
-        }
-    }
-
-    public function initialize_option() {
-        if ( !get_option( 'rtsyntax_options' ) ) {
-            $options = array( 'theme' => 'default' );
-            add_option( 'rtsyntax_options', $options );
-        }
-    }
-
-    public function register_settings() {
- 	add_settings_section( 'rtsyntax-options', __( 'Code Theme', 'rtSyntax' ), array( $this, 'settings_section' ), 'rtsyntax' );
- 	add_settings_field( 'rtsyntax-theme', __( 'Theme', 'rtSyntax' ), array( $this, 'settings_field' ), 'rtsyntax', 'rtsyntax-options' );
-        register_setting( 'rtsyntax', 'rtsyntax_options' );
-    }
-
-    public function settings_section(){
-    }
-
-    public function settings_field() {
-        $options = get_option( 'rtsyntax_options' ); ?>
-        <select id="rtsyntax-theme" name="rtsyntax_options[theme]">
+/**
+ * Entry File For Our Plugin.
+ *
+ * @package rtsyntax
+ */
+
+/**
+ * Plugin Name: rtSyntax
+ * Plugin URI: http://rtcamp.com
+ * Author: rtCamp
+ * Author URI: http://rtcamp.com
+ * Version: 1.0.5
+ * Description: A no-fuss, lightweight, fast and optimised syntax highlighter for WordPress. Tested upto 4.9.4
+ * Contributors: rtcamp, rahul286, JoshuaAbenazer
+ * Tags: code highlighter, highlighter, highlighting, syntax, syntax highlighter, source, jquery, javascript, nginx, php, code, CSS, html, php, sourcecode, xhtml, languages, TinyMCE
+ */
+
+/**
+ * Class rtSyntax
+ */
+class RtSyntax {
+
+	/**
+	 * RtSyntax constructor.
+	 */
+	public function __construct() {
+		register_activation_hook( __FILE__, array( $this, 'initialize_option' ) );
+		if ( is_admin() ) {
+			add_action( 'admin_init', array( &$this, 'register_settings' ) );
+			add_action( 'admin_menu', array( &$this, 'admin' ) );
+
+			add_filter( 'mce_external_plugins', array( $this, 'rtsyntax_buttons' ) );
+			add_filter( 'mce_buttons', array( $this, 'register_rtsyntax_buttons' ) );
+		} else {
+			add_action( 'wp_enqueue_scripts', array( $this, 'enqueue' ) );
+			add_action( 'wp_head', array( $this, 'onload' ) );
+			add_action( 'the_content', array( $this, 'convert_pres' ) );
+		}
+	}
+
+	/**
+	 * Initialize plugin option
+	 */
+	public function initialize_option() {
+		if ( ! get_option( 'rtsyntax_options' ) ) {
+			$options = array( 'theme' => 'default' );
+			add_option( 'rtsyntax_options', $options );
+		}
+	}
+
+	/**
+	 * Register settings in settings section
+	 */
+	public function register_settings() {
+		add_settings_section( 'rtsyntax-options', __( 'Code Theme', 'rtSyntax' ), array( $this, 'settings_section' ), 'rtsyntax' );
```

Comment:
Change text-domain name from `rtSyntax` to `rtsyntax` at all places. Text domain slug should similar to plugin slug and in lower case.